### ISSUE TITLE
[Druid] Eclipse mechanic

### ISSUE THREAD

**Issue by Aveta** [Valhalla-Gaming/Tracker#4987]
After a recent update I'm permamently staying in Lunar Eclipse. Even when my bar is in Solar, Moonfire doesn't switch and I get Lunar peaks constantly. No respecing or cache clearing helped.
Also Lunar empowerment stacks granted by Starsurge aren't properly consumed.

**Comment by Moustachos** [comment]
Couldn't reproduce your issue of getting stuck in Lunar Eclipse, looks like an isolated bug.

However, the Lunar / Solar Empowerment stacks issue will be fixed on next update.

**Comment by Moustachos** [comment]
Ref #2266

**Comment by Aveta** [comment]
Currently I do get Solar Eclipse, but only once every few cycles. I'll open a new issue if I get to record a video. I tried deleting both cache and WTF folders, resetting spec completely, checked if any glyph/talent was the cause but everytime it was the same thing. Maybe I get some weird delay, no idea.
